pegasus/sites.v3/code.org/public/pd-workshop-survey/materials/splat.haml

```diff
@@ -0,0 +1,105 @@
+  pass unless enrollment
+
+  # Make sure it's a CSF workshop
+  workshop = DASHBOARD_DB[:pd_workshops].where(id: enrollment[:pd_workshop_id]).first
+  pass unless workshop && workshop[:course] == 'CS Fundamentals'
+
+  # Make sure they completed the survey first
+  survey_response = DB[:forms].where(kind: 'PdWorkshopSurvey', source_id: enrollment[:id]).first
+  pass unless survey_response
+
+  # Have they already filled out this form?
+  previous_response = DB[:forms].where(kind: 'PdWorkshopMaterials', source_id: enrollment[:id]).first
+
+%script{type: "text/javascript", src: "/js/pd-workshop-materials.js"}
+%link{rel: "stylesheet", type: "text/css", href: "/css/selectize.bootstrap3.css"}/
+
+- if previous_response
+
+  %h1 Thank you
+  %h3 Your request has been received. Please allow between 4-5 weeks for your materials to arrive.
+
+- else
+
+  %form#pd-workshop-materials-form{role: 'form'}
+    %input{name: 'enrollment_id_i', type:'hidden', value: enrollment[:id]}
+    %input{name: 'workshop_id_i', type: 'hidden', value: workshop[:id]}
+
+    %h1 Thank you for your survey submission
+
+    %h2
+      Complete the following form if you'd like to receive materials to teach the
+      unplugged lessons in Courses 1-4 (US attendees only).
+
+    .form-group
+      %label.control-label{for: 'name_s'}
+        Your Name
+        %span.form-required-field *
+      %input.form-control{name: 'name_s', type: 'text'}
+
+    .form-group
+      %label.control-label{for: 'school_s'}
+        School/Company Name (optional)
+      %input.form-control{name: 'school_s', type: 'text'}
+
+    .form-group
+      %label.control-label{for: 'address_1_s'}
+        Street Address 1
+        %span.form-required-field *
+      %input.form-control{name: 'address_1_s', type: 'text'}
+
+    .form-group
+      %label.control-label{for: 'address_2_s'}
+        Street Address 2 (optional)
+      %input.form-control{name: 'address_2_s', type: 'text'}
+
+    .form-group
+      %label.control-label{for: 'city_s'}
+        City
+        %span.form-required-field *
+      %input.form-control{name: 'city_s', type: 'text'}
+
+    .form-group
+      %label.control-label{for: "state_s"}<
+        State
+        %span.form-required-field *
+      %div
+        %select#k5-send-materials-state.form-control{name: "state_s"}
+          %option{selected: true, value: ""}
+          -STATE_ABBR_WITH_DC_HASH.each_pair do |abbr, state|
+            %option{value: abbr}= state
+
+    .form-group
+      %label.control-label{for: 'zip_code_s'}
+        Zip Code
+        %span.form-required-field *
+      %input.form-control{name: 'zip_code_s', type: 'text'}
+
+    .form-group
+      %label.control-label{for: 'email_s'}
+        Email
+        %span.form-required-field *
+      %input.form-control{name: 'email_s', type: 'text'}
+
+    .form-group
+      %label.control-label{for: 'phone_number_s'}
+        Phone Number
```

Comment:
Not sure what it looks like in the view but could there be a little suggestion/placeholder text like XXX-XXX-XXXX or 123-456-7890? Even though we'll be smart to accept pretty much any format they try, might be easier to be directed to think one way?